spot_rl_experiments/experiments/comparisons/gaze_all_objects.py

```diff
@@ -28,10 +28,15 @@ def main(spot, bd=False):
         config.MAX_EPISODE_STEPS = 150
 
     spot.power_robot()
-    gaze_controller = GazeController(config, spot)
+    gaze_controller = Pick(config, spot, use_mobile_pick=False)
```

Comment:
That's a good point. I made it Pick because pick.execute(target) seemed more natural than gaze.execute(target)
But our policy & env are still called Gaze, seems like it would be a better idea to simply call it Gaze rather than Pick.

Reply:
@akshararai  @zephirefaith  should I change the class name to `Gaze` ?? Or `Pick` is okay?

Reply:
As discussed offline, `Pick` name for class is okay.